.circleci/config.yml

```diff
@@ -353,19 +328,10 @@ workflows:
             branches:
               ignore:
               - /docs-preview/
-      - build-bazel:
-          requires:
-            - build
       - test:
           requires:
             - build
-      - test-large:
-          requires:
-            - build
-      - test-large:
-          name: test-large-ve
-          ve: true
-          glob: "packages/angular_devkit/@(build_angular|build_ng_packagr)/@(src|test)/@(build|browser)/*_spec_large.ts"
+      - integration:
           requires:
             - build
```

Comment:
Same for integration?

Reply:
I'm following the same logic as above, since the integration test also builds sources.